### PR TITLE
[Windows] [PT2.6] Add MSVC flag

### PR DESCRIPTION
**Pull request by ratnampa**

### CODE REVIEW

File: src/BuildOnWindows.cmake [hunk 1]
```diff
@@ -24,6 +24,7 @@ add_library(
   ${ATen_XPU_GEN_SRCS})
 install(TARGETS torch_xpu_ops_aten DESTINATION "${TORCH_INSTALL_LIB_DIR}")
 # target_compile_definitions(torch_xpu_ops_aten PRIVATE CAFFE2_BUILD_MAIN_LIB)
+target_compile_options(torch_xpu_ops_aten PRIVATE /permissive-)
```
Comment (fengyuan14) [inline src/BuildOnWindows.cmake:27]: What's the usage? Could you elaborate the add in the description of the commit.

**Comment by fengyuan14** [comment]
Should be a duplicate PR. https://github.com/intel/torch-xpu-ops/pull/943 I saw the related issue. Thanks. I will close the PR, once  https://github.com/intel/torch-xpu-ops/pull/943 is merged.

**Comment by fengyuan14** [comment]
> Should be a duplicate PR. #943 I saw the related issue. Thanks. I will close the PR, once #943 is merged.

I checked the internal JIRA of the issue. Compiler team said it is a WA for the building error. Could you describe the issue and mark the change as a WA?

**Comment by ratnampa** [comment]
@fengyuan14 the MSVC compiler faces issues with same namespace std being used in the code and this flag allows for stricter conformance and it is able to compile without running into **ambiguous symbol std error**

**Comment by ratnampa** [comment]
@fengyuan14 I no longer run into this error when building PT2.6 master with oneAPI 2025.0 dl-essentials-bundle. For now I can close this and re-open if the same error occurs.

**Pull request closed without merging**